docs/starter-kit-bot-details.md

```diff
@@ -0,0 +1,84 @@
+# Starter Kit Bot Details
```

Comment:
do the anchors work from the individual kit pages?

Reply:
if you're referring to the `bot details` link, yep it works. I tested it locally.